x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -658,7 +677,15 @@ export const UserProfile: FunctionComponent<UserProfileProps> = ({ user, data })
                   user={user}
                   onShowPasswordForm={() => setShowChangePasswordForm(true)}
                 />
-                {isCloudUser ? null : <UserSettingsEditor formik={formik} />}
+                {isCloudUser ? null : (
+                  <UserSettingsEditor
+                    formik={formik}
+                    isDarkModeOverride={determineIfDarkModeOverride(
+                      isThemeOverridden,
+                      isOverriddenThemeDarkMode
+                    )}
```

Comment:
nit: Maybe pass `services.settings.client` directly as an argument into `determineIfDarkModeOverride()` instead of the two booleans so that that function handles all of the complexity of determining if dark mode is overridden?